grpc-ingest/src/prom.rs

```diff
@@ -258,6 +267,16 @@ pub fn grpc_tasks_total_dec(label: &str, stream: &str) {
     GRPC_TASKS.with_label_values(&[label, stream]).dec()
 }
 
+pub fn inc_download_metadata_inserted_count() {
+    DOWNLOAD_METADATA_INSERTED_COUNT.inc();
+}
+
+pub fn download_metadata_json_failed_task_inc(status: u16) {
+    DOWNLOAD_METADATA_FAILED_COUNT
+        .with_label_values(&[&status.to_string()])
+        .inc();
+}
```

Comment:
I just had the thought is that we can have just the inserted metric and then use status 200. And metric can be `download_metadata_fetched_count{status}`